target/linux/ramips/dts/mt7621_xiaomi_mi-router-4.dts

```diff
@@ -0,0 +1,198 @@
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "xiaomi,mi-router-4", "mediatek,mt7621-soc";
+	model = "Xiaomi Mi Router 4";
+
+	aliases {
+		led-boot = &led_status_yellow;
+		led-failsafe = &led_status_red;
+		led-running = &led_status_blue;
+		led-upgrade = &led_status_yellow;
+		label-mac-device = &gmac0;
```

Comment:
Have you verified that MAC address label corresponds to LAN MAC address?

Reply:
I've checked this is correct

Reply:
Yes, verified here https://github.com/openwrt/openwrt/pull/3649#discussion_r536383887